_includes/footer.html

```diff
@@ -55,20 +55,20 @@
       <ul class="usa-identifier__required-links-list">
         <li class="usa-identifier__required-links-item">
           <a
-            href="https://home.dotgov.gov/about/"
+            href="../../about"
             class="usa-identifier__required-link usa-link"
             >About .gov</a
           >
         </li>
         <li class="usa-identifier__required-links-item">
           <a
-            href="https://github.com/cisagov/getgov"
+            href="https://github.com/cisagov/getgov-home"
             class="usa-identifier__required-link usa-link usa-link--external"
             >.gov on Github</a
           >
         </li>
         <li class="usa-identifier__required-links-item">
-          <a href="https://home.dotgov.gov/privacy/" class="usa-identifier__required-link usa-link"
+          <a href="TODO" class="usa-identifier__required-link usa-link"
```

Comment:
So far I have been making all of the placeholder links `href` just "#", and those are the ones that automatically get the yellow highlight and "TBD" text. I am ok with switching them to "TODO" instead but let's just coordinate.